docs/content/preview/troubleshoot/cluster/failed_tablets.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Recover YB-TServer from crash loop
+linkTitle: Recover YB-TServer from crash loop
+description: Recover YB-TServer from crash loop
+menu:
+  preview:
+    parent: troubleshoot-cluster
+    weight: 866
+isTocNested: true
+showAsideToc: true
+---
+
+When a YB-TServer process or node has failed, YugabyteDB automatically triggers a remote bootstrap for
+ most types of tablet data corruption or failures. However, there is still a number of cases a number of cases where this automatic mechanism might not be able to help.
+
+One major cause of such case would be a condition that leads to a code path encoded as a crash (for example, `CHECK` or `FATAL`) or the causes of a crash is unknown (for example, code errors that lead to `SIGSEGV`). 
+In all of these cases, the root cause would likely repeat itself when the process is restarted.
+Moreover, when using YugabyteDB Anywhere, crashed processes are automatically restarted to ensure minimum downtime.
+
+Since servers stuck in a crash loop typically cannot stay up long enough to be safely issued runtime commands against them, manual intervention by the administrator is required to bring a YB-TServer back to a healthy state.
+
+To do this, the administrator needs to find all the faulty tablets, look for their data on disk (possibly spread across multiple disks, depending on your `fs_data_dirs`), and then remove it.
+The following are the steps to address this scenario:
+
+1. Stop the YB-TServer process to prevent new restarts, during operations. For YugabyteDB Anywhere, execute the yb-server-ctl tserver stop command.
```

Comment:
execute the yb-server-ctl tserver stop command.  --- Code element formatting is lost?